lib/langchain/messages/chat_message.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Langchain
+  module Messages
+    # Generic messages that can be used in the `chat` method of the LLM adapter
+    # It is recommended that `assistants/messages/*_messages.rb` be unified with this one in the future.
+    class ChatMessage
```

Comment:
Does it make sense to simplify the namespace structure to just `Langchain::ChatMessage`?

Reply:
I had placed it in a namespace aligned with [lib/langchain/assistants/messages/openai_message.rb](https://github.com/patterns-ai-core/langchainrb/blob/main/lib/langchain/assistants/messages/openai_message.rb). I think it should be fine to put it at the root level.